ext/POSIX/lib/POSIX.pod

```diff
@@ -825,11 +825,12 @@ to use instead.  See L<perlrecharclass/POSIX Character Classes>.
 Returns true if the argument is C<NaN> (not-a-number) [C99].  Added in
 Perl v5.22.
 
-Note that you cannot test for "C<NaN>-ness" with
+Note that you can also test for "C<NaN>-ness" with
+L<equality operators|perlop/"Equality Operators"> (C<==> or C<!=>), as in
 
-   $x == $x
+    print "x is not a NaN\n" if $x == $x;
```

Comment:
consider promoting `say` instead of `print`?

Reply:
I'd like to use `say` too, but I think it is a bit premature to use it here because it is currently not enabled by default (adding such as `use v5.10;` might be confusing in this one-line code example), and the rest of POSIX.pod (and most of core documentations) still use `print`.